packages/web/public/static/locales/en/download.json

```
@@ -1,11 +1,11 @@
 {
-  "coverTitle": "Celo Test Wallet",
-  "coverSubTitle": "Temporarily Not Available",
+  "coverTitle": "Celo Developer Wallet",
+  "coverSubTitle": "A testing wallet.\nDo not send real funds.",
```

actually can we update this to be more descriptive?

```suggestion
  "coverSubTitle": "An Alfajores Testnet wallet.\nDo not send real funds.",
```